tensorflow/python/ops/array_ops.py

```diff
@@ -125,6 +125,32 @@ def shape(input, name=None):
     else:
       return gen_array_ops.shape(input, name=name)
 
+def size(input, name=None):
+  """Returns the size of a tensor.
+
+  This operation returns an integer representing the number of elements in
+  `input`.
+
+  For example:
+
+  ```python
+  # 't' is [[[1, 1,, 1], [2, 2, 2]], [[3, 3, 3], [4, 4, 4]]]]
```

Comment:
I see that the double-comma typo has managed to live its life for a while in our official doc...Mind fixing it while you're at it?